lib/WasmReader/ModuleInfo.cpp

```diff
@@ -19,30 +19,21 @@ ModuleInfo::ModuleInfo(ArenaAllocator * alloc) :
 }
 
 bool
-ModuleInfo::InitializeMemory(size_t minSize, size_t maxSize, bool exported)
+ModuleInfo::InitializeMemory(uint32 minPage, uint32 maxPage, bool exported)
 {
     if (m_memory.minSize != 0)
```

Comment:
Is this a stale check? m_memory has an initializer for minSize(0). Is this trying to indirectly ensure that memory isn't initialized twice?